api/_utils/db/db-manager.test.ts

```diff
@@ -404,4 +407,28 @@ describe("db-manager", () => {
       `"tree _R447NTgmyk was unadopted by user auth0|TDilMAYKVk"`,
     );
   });
+
+  test("unwaterTree", async () => {
+    await client.query("INSERT INTO trees (id) VALUES ($1)", ["_08be12a72n"]);
```

Comment:
could this not be done in one call like the one below using `format`

Reply:
I guess it could. I will try. I just copied it from the test before. If it works I will apply it above as well.